app/models/content/content_item.rb

```diff
@@ -0,0 +1,56 @@
+module Content
+  class ContentItem < ApplicationRecord
```

Comment:
I think that is okay. There is always the option of fully qualifying it with `::Content::Item` where just `Item` might be confusing.